module/GameMotion/GameMotion.h

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file BingoMotion.h
+ * @brief ビンゴ動作の親クラス
+ * @author kodama0720
+ */
+
+#ifndef BINGO_MOTION_H
+#define BINGO_MOTION_H
+
+class GameMotion {
+ public:
+  /**
+   * コンストラクタ
+   */
+  GameMotion(double time = 0, double risk = 0);
```

Comment:
デフォルト値を設定しない方が，派生先で値を設定し忘れることを予防できませんか？
```suggestion
  GameMotion(double time, double risk);
```